gui/CMakeLists.txt

```diff
@@ -149,6 +156,10 @@ target_link_libraries(mapgui
   ${QT_LIBRARIES}
 )
 
+if(vtkWebGLExporter_LOADED)
+  target_link_libraries(mapgui vtkWebGLExporter)
```

Comment:
Ah, right, because moc needs the definitions when it looks for slots. (Alternatively, you could work around that by making the slot not conditional but possibly empty-bodied.)